spec/controllers/api/v1/cx_responses_controller_spec.rb

```diff
@@ -57,6 +57,8 @@
         end
 
         it 'return a default paginated array of cx_responses' do
+          expect(cx_collection_detail.cx_responses.count).to eq(1000)
```

Comment:
ensures these 1000 records exist first.

the 1000 records are loaded in this text as cx_responses, from a CSV fixture (seed file)